content/applications/sales/sales/send_quotations/create_quotations.rst

```diff
@@ -0,0 +1,332 @@
+default, there are a collection of quotation and order settings that can be customized and
+configured, if needed.
+
+To access these setting options, navigate to :menuselection:`Sales app --> Configuration -->
+Settings`, and scroll to the :guilabel:`Quotations & Orders` section.
+
+.. image:: create_quotations/quotations-orders-section.png
+   :align: center
+   :alt: The Quotations and Orders section on the Odoo Sales app Settings page.
+
+- :guilabel:`Quotation Templates`: create unique quotation templates featuring standardized offers
+  with default products, which can be easily accessed from a quotation form. When this checkbox is
+  ticked, an additional field, :guilabel:`Default Template` appears, along with a link to the
+  :guilabel:`Quotation Templates` page. To learn more, refer to the :doc:`Quotation templates
+  <quote_template>` documentation.
+- :guilabel:`Online Signature`: request an online signature to confirm orders. To learn more, refer
+  to the :doc:`Online payment order confirmation <get_signature_to_validate>` documentation.
+- :guilabel:`Online Payment`: request an online prepayment from customers to confirm orders. Request
+  a full payment or paritial payment (via down payment). When this checkbox is ticked, an additional
+  field, :guilabel:`Prepayment amount (%)`, appears. There is also a link to the :guilabel:`Payment
+  Providers` page, as well. To learn more, refer to the :doc:`Online payment order confirmation
+  <get_paid_to_validate>` documentation.
+- :guilabel:`Default Quotation Validity`: determine an amount (in :guilabel:`days`) that quotations
+  can remain valid for, by default.
+- :guilabel:`Default Recurrence`: determine a default period from the drop-down menu to use as a
+  recurrence for a new quotation.
+- :guilabel:`Sale Warnings`: get warnings in orders for specific products or customers.
+- :guilabel:`PDF Quote builder`: customize the look of quotations with unique header pages, product
+  descriptions, footer pages, and more. To learn more, refer to the :doc:`PDF quote builder
+  <pdf_quote_builder>` documentation.
+- :guilabel:`Lock Confirmed Sales`: ensure that no further edits can be made to confirmed orders.
+- :guilabel:`Pro-Forma Invoice`: send pro-forma invoices to customers. To learn more, refer to the
+  :doc:`Pro-forma invoices <../invoicing/proforma>` documentation.
+
+To activate any of these settings, tick the checkbox beside the desired option(s). Then, click
+:guilabel:`Save`.
+
+Quotations dashboard
+====================
+
+To access the *Quotations* dashboard, which is the default dashboard of the *Sales* app, simply
+click the :menuselection:`Sales app` from the main Odoo dashbord.
+
+.. tip::
+   If already in the *Sales* application, the :guilabel:`Quotations` dashboard can be accessed by
+   navigating to :menuselection:`Orders --> Quotations` from the header menu in the *Sales*
+   application.
```

Comment:
This is a great question. And, frankly, I am not entirely certain. Just to explain my stance, I feel like, since I'm **only** mentioning *if* they are in the application (and not directing them to navigate to/open the app), I opted for the use of italics, instead of the menuselection.

However, I fully see your point, and understand this a bit of an "unorthodox" way to format such a sentence/direction.

So, since this is a kind of "up in the air" issue, I am going to choose to leave it as-is for now, but if ZST or SALI think a different approach should be taken, I'll be sure to make the necessary adjustment. Thanks for bringing this to my attention, though. Great eye for detail 👁️ 👍